ChurchToolsApi.py

```diff
@@ -538,6 +538,41 @@ def get_event_agenda(self, event_id: int):
             logging.info("Event requested that does not have an agenda with status: {}".format(response.status_code))
             return None
 
+    def export_event_agenda(self, target_format, agenda_id, target_path):
```

Comment:
please set a default for target path e.g. 
`path_for_download='./Downloads' `
similar to file_download

Reply:
I've added a default path. Moreover it is possible to define the exact filename or only a foldername (then Defaultfilename is used) for the download now.